scripts/install.sh

```diff
@@ -5,7 +5,7 @@ HZCLI_HOME="$HOME/.local/share/hz-cli"
 
 ghExtractTag() {
   tagUrl=$(curl "https://github.com/$1/releases/latest" -s -L -I -o /dev/null -w '%{url_effective}')
-  printf "%s\n" "${tagUrl##*v}"
```

Comment:
`x="https://github.com/hazelcast/hazelcast-commandline-client/releases/tag/vv0.1.0-preview.1"`

But it will also parse as if it corrects, does it create a worry ?

Reply:
I guess we are retrieving URL from the `curl`, That case will not be happened.

Reply:
Yes this assumes we will use semver for all tags.